bitcoin/exchange/bitcoinprices.go

```diff
@@ -89,11 +96,12 @@ func (b *BitcoinPriceFetcher) fetchCurrentRates() error {
 }
 
 type BitcoinAverage struct {
-	cache map[string]float64
+	cache  map[string]float64
+	client *http.Client
```

Comment:
For the prices, I would like to standardize the API response there and have a list of price servers (maybe OB1, DUO) that all return the same thing rather than have to handle responses from multiple sources. 

I think that's a much better approach than this.